pymc/tests/test_distributions.py

```diff
@@ -2136,6 +2137,42 @@ def test_dirichlet_2D(self):
             dirichlet_logpdf,
         )
 
+    @pytest.mark.parametrize(
+        "value,alpha,K,logp",
+        [
+            (np.array([5, 4, 3, 2, 1]) / 15, 0.5, 4, 2.2057773112876893),
+            (np.tile(1, 13) / 13, 2, 12, 13.286898065112881),
+            (np.array([0.001] * 10 + [0.99]), 0.1, 10, -20.66907735582068),
+            (np.append(0.5 ** np.arange(1, 20), 0.5 ** 20), 5, 19, 92.59518981534681),
+        ],
+    )
+    def test_stickbreakingweights(self, value, alpha, K, logp):
+        with Model() as model:
+            StickBreakingWeights("sbw", alpha=alpha, K=K)
+        pt = {"sbw": value}
+        assert_almost_equal(
+            model.fastlogp(pt),
+            logp,
+            decimal=select_by_precision(float64=6, float32=2),
+            err_msg=str(pt),
+        )
```

Comment:
I drew inspiration `test_ex_gaussian` because there is no scipy or numpy logp equivalent. Here is the error that I'm getting with this test.

<img width="1271" alt="Screen Shot 2021-11-17 at 5 42 05 PM" src="https://user-images.githubusercontent.com/24764859/142294239-8b8e9bb2-2b21-4132-8d9d-e55876f3852a.png">

Reply:
Solved by using `transform=None`, which is okay when we're just testing the logp method